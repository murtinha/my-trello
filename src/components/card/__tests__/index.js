import React from 'react';
import TestRenderer from 'react-test-renderer';
import Card from '../';


describe('<Card />', () => {
  describe('render', () => {
    it('should render properly', () => {
      const comp = TestRenderer
       .create(<Card description="ola" />)
       .toJSON();
      expect(comp).toMatchSnapshot();
    });
  });
});
