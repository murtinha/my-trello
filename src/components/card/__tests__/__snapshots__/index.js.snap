// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> render should render properly 1`] = `
<div
  className="Card"
>
  <div
    className="Card-description"
  >
    ola
  </div>
  <div
    className="Card-footer"
  />
</div>
`;
